### PR TITLE
[13.0][FIX] partner_company_group: assure no error in migration

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Just in case you already have partner_company_group in previous versions.

### CODE REVIEW

File: partner_company_group/migrations/13.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -9,4 +9,12 @@
...
 def migrate(env, version):
-    openupgrade.rename_fields(env, field_renames)
+    if openupgrade.column_exists(env.cr, "res_partner", "group_id"):
+        if not openupgrade.column_exists(env.cr, "res_partner", "company_group_id"):
+            openupgrade.rename_fields(env, field_renames)
+        else:
+            openupgrade.logged_query(
+                env.cr, """
+                UPDATE res_partner
+                SET company_group_id = group_id
```
Comment (pedrobaeza) [inline partner_company_group/migrations/13.0.1.0.0/pre-migration.py:20]: The target model is the same?

Comment (MiquelRForgeFlow) [reply partner_company_group/migrations/13.0.1.0.0/pre-migration.py:20]: yes, both are res.partner

Comment (MiquelRForgeFlow) [reply partner_company_group/migrations/13.0.1.0.0/pre-migration.py:20]: I could add a query to check both field target same model, but that would make dirty the code

Comment (pedrobaeza) [reply partner_company_group/migrations/13.0.1.0.0/pre-migration.py:20]: OK, just want to make sure of that.

**Comment by pedrobaeza** [comment]
Please check pre-commit

**Comment by MiquelRForgeFlow** [comment]
Updated :)

**Review by pedrobaeza — APPROVED** [review]

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**